build-tools/packages/build-cli/src/lib/package.ts

```diff
@@ -33,53 +46,86 @@ import { isReleaseGroup, ReleaseGroup, ReleasePackage } from "../releaseGroups";
 // eslint-disable-next-line max-params
 export async function npmCheckUpdates(
     context: Context,
-    releaseGroup: ReleaseGroup | ReleasePackage,
+    releaseGroup: ReleaseGroup | ReleasePackage | undefined,
     depsToUpdate: ReleasePackage[] | RegExp[],
-    bumpType: "patch" | "minor" | "current",
+    releaseGroupFilter: ReleaseGroup | undefined,
+    depUpdateType: DependencyUpdateType,
     // eslint-disable-next-line default-param-last
     prerelease = false,
     // eslint-disable-next-line default-param-last
     writeChanges = false,
     log?: Logger,
 ): Promise<{
     updatedPackages: Package[];
-    updatedDependencies: Package[];
+    updatedDependencies: PackageVersionMap;
 }> {
     const updatedPackages: Package[] = [];
-    const deps = new Set<string>();
+
+    /** A set of all the packageName, versionString pairs of updated dependencies. */
+    const updatedDependencies: PackageVersionMap = {};
+
     // There can be a lot of duplicate log lines from npm-check-updates, so collect and dedupe before logging.
     const upgradeLogLines = new Set<string>();
     const searchGlobs: string[] = [];
     let repoPath: string;
 
     log?.info(`Checking npm for package updates...`);
-    if (isReleaseGroup(releaseGroup)) {
+    if (releaseGroup === undefined) {
+        // Apply to all packages in repo
+        repoPath = context.repo.resolvedRoot;
+
+        for (const monorepo of context.repo.releaseGroups.values()) {
```

Comment:
Nit: "currentReleaseGroup" (or something) instead of "monorepo"?

Reply:
Went with `releaseGroupRoot`.